.github/workflows/test.yml

```diff
@@ -104,6 +104,7 @@ jobs:
             9.0.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
             9.2.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
             9.4.4) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-unstable ;;
+            9.6.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-unstable ;;
```

Comment:
Both 9.6.2 and 9.4.4 can be obtained from `nixos-23.05` instead of `nixos-unstable`.  Since the latter is a moving target, we should use the former for these now that it is available.